cloudant-client/src/test/java/com/cloudant/tests/base/TestWithDb.java

```diff
@@ -0,0 +1,30 @@
+package com.cloudant.tests.base;
+
+import com.cloudant.client.api.CloudantClient;
+import com.cloudant.client.api.Database;
+import com.cloudant.tests.extensions.CloudantClientExtension;
+import com.cloudant.tests.extensions.DatabaseExtension;
+import com.cloudant.tests.extensions.MultiExtension;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+// Base class for tests which require a DB which is used throughout the lifetime of the test class
+public class TestWithDb {
```

Comment:
What about having a `TestWithDbPerClass` that would extend a base `TestWithDb ` class? 
`TestWithDbPerTest` would also extend `TestWithDb`.

Reply:
@emlaver good idea, done in fe8add3